tests/fixtures/attributes/attributes.yml

```diff
@@ -1,67 +1,191 @@
 -
-  actual:
-    class:
-      'a simple string'
-  expected:
-    ' class="a simple string"'
+  given:
+    class: ['one', 'two']
+  run:
+    - append: ['class', 'three']
+    - remove: ['class', 'two']
+  expect:
+    - toArray:
+        return:
+          class: ['one', 'three']
+    - __toString:
+        return: ' class="one three"'
```

Comment:
I don't know what these fixture changes have to do with the Leo / peridot assertions.
I imagine they will conflict with the Attributes PR.

Reply:
It's just a file rename.